store/copr/batch_request_sender.go

```diff
@@ -100,22 +98,18 @@ func (ss *RegionBatchRequestSender) onSendFailForBatchRegions(bo *Backoffer, ctx
 		return tikverr.ErrTiDBShuttingDown
 	}
 
-	if config.GetGlobalConfig().DisaggregatedTiFlash {
```

Comment:
Delete this if branch because there will be no batchCop protocol in disaggregated tiflash mode. Also mpp error handling will be handled in `mpp.go`